src/test/java/org/myrobotlab/service/HttpClientTest.java

```diff
@@ -35,36 +30,17 @@ public void testService() throws Exception {
 
       // curl --location --request POST "https://postman-echo.com/post" --data
       // "foo1=bar1&foo2=bar2"
-      http.postForm("https://postman-echo.com/post", "foo1", "bar1", "foo2", "bar2", "foo3", "bar with spaces");
-      log.info(data);
+      String test = http.postForm("https://postman-echo.com/post", "foo1", "bar1", "foo2", "bar2", "foo3", "bar with spaces");
```

Comment:
bad test, we shouldn't be calling out to an external website from our unit tests.